src/openzeppelin/token/erc20/presets/ERC20.cairo

```diff
@@ -10,140 +10,102 @@ from starkware.cairo.common.uint256 import Uint256
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }(owner: felt, spender: felt) -> (remaining: Uint256):
-    let (remaining: Uint256) = ERC20.allowance(owner, spender)
-    return (remaining)
-end
-
-#
-# Externals
-#
+func allowance{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    owner: felt, spender: felt
+) -> (remaining: Uint256) {
+    let (remaining: Uint256) = ERC20.allowance(owner, spender);
+    return (remaining,);
+}
+
+//
+// Externals
+//
 
 @external
-func transfer{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }(recipient: felt, amount: Uint256) -> (success: felt):
-    ERC20.transfer(recipient, amount)
-    return (TRUE)
-end
+func transfer{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    recipient: felt, amount: Uint256
+) -> (success: felt) {
+    ERC20.transfer(recipient, amount);
+    return (TRUE,);
+}
 
 @external
-func transferFrom{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }(
-        sender: felt,
-        recipient: felt,
-        amount: Uint256
-    ) -> (success: felt):
-    ERC20.transfer_from(sender, recipient, amount)
-    return (TRUE)
-end
+func transferFrom{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    sender: felt, recipient: felt, amount: Uint256
+) -> (success: felt) {
+    ERC20.transfer_from(sender, recipient, amount);
+    return (TRUE,);
+}
 
 @external
-func approve{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }(spender: felt, amount: Uint256) -> (success: felt):
-    ERC20.approve(spender, amount)
-    return (TRUE)
-end
+func approve{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    spender: felt, amount: Uint256
+) -> (success: felt) {
+    ERC20.approve(spender, amount);
+    return (TRUE,);
+}
 
 @external
-func increaseAllowance{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }(spender: felt, added_value: Uint256) -> (success: felt):
-    ERC20.increase_allowance(spender, added_value)
-    return (TRUE)
-end
+func increaseAllowance{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    spender: felt, added_value: Uint256
+) -> (success: felt) {
+    ERC20.increase_allowance(spender, added_value);
+    return (TRUE,);
+}
 
 @external
-func decreaseAllowance{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }(spender: felt, subtracted_value: Uint256) -> (success: felt):
-    ERC20.decrease_allowance(spender, subtracted_value)
-    return (TRUE)
-end
+func decreaseAllowance{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    spender: felt, subtracted_value: Uint256
+) -> (success: felt) {
+    ERC20.decrease_allowance(spender, subtracted_value);
+    return (TRUE,);
```

Comment:
all these methods should return the library method's return value

Reply:
instead of crafting their own `TRUE`

Reply:
On it!